### ISSUE TITLE
problem when meteor run?

### ISSUE THREAD

**Issue by thearabbit** [hitchcott/app-dump#4]
Problem when meteor run:

```
Your app is crashing. Here's the latest log.


/home/theara/.meteor/packages/meteor-tool/.1.0.41.xwwsz8++os.linux.x86_64+web.browser+web.cordova/meteor-tool-os.linux.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:173
                        throw(ex);
                              ^
Error: Can't find npm module 'Busboy'. Did you forget to call 'Npm.depends' in package.js within the 'hitchcott_app-dump' package?
    at Object.Npm.require (/home/theara/Desktop/meteor/rabbit/.meteor/local/build/programs/server/boot.js:147:17)
    at Package (packages/hitchcott:app-dump/app-dump-server.coffee:4:14)
    at packages/hitchcott:app-dump/app-dump-server.coffee:1:1
    at packages/hitchcott:app-dump/app-dump-server.coffee:1:1
    at /home/theara/Desktop/meteor/rabbit/.meteor/local/build/programs/server/boot.js:205:10
    at Array.forEach (native)
    at Function._.each._.forEach (/home/theara/.meteor/packages/meteor-tool/.1.0.41.xwwsz8++os.linux.x86_64+web.browser+web.cordova/meteor-tool-os.linux.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
    at /home/theara/Desktop/meteor/rabbit/.meteor/local/build/programs/server/boot.js:116:5
Exited with code: 8
Your application is crashing. Waiting for file change.
```

**Comment by IstoraMandiri** [comment]
Sorry, I forgot to publish this to atmosphere. It was fixed in 0.1.3!

Please try again now, make sure you're using 0.1.3. Cheers.

**Comment by thearabbit** [comment]
Thanks I will try again.
